curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-describe-your-current-project/655be33b7a463a5593c91cb4.md

```diff
@@ -0,0 +1,45 @@
+---
+id: 655be33b7a463a5593c91cb4
+title: Task 21
+challengeType: 22
+dashedName: task-21
+audioPath: curriculum/js-music-player/We-Are-Going-to-Make-it.mp3
+---
+<!--
+AUDIO REFERENCE: 
+Sarah: Pictures can be great to explain things. We are not using pictures, but it's surely fun to work with data like this.
+-->
+
+# --description--
+
+`Sure` means to be confident or certain about something. 
+
+When you add the suffix `-ly`, `sure` becomes an adverb, `surely`, which means doing something in a certain or confident way. 
+
+It's often used to emphasize what the speaker believes is true or likely to be true. For example:
+
+`Surely, you can see why this is important` (emphasizing that the importance is expected to be clear)
+
+`She will surely finish the project on time` (showing confidence in her completing the project)
+
+# --fillInTheBlank--
+
+## --sentence--
+
+`Pictures can be _ to explain things. We are not using pictures, but it's _ fun to work with data like this.`
```

Comment:
@naomi-lgbt I see what you mean. Unfortunately, we won't be able to change that because this line is based on the audio Quincy and Abbey recorded. So, we'll have too keep it this way.